app/controllers/items_controller.rb

```diff
@@ -2,7 +2,8 @@ class ItemsController < ApplicationController
   def index
     if params[:merchant_id]
       @merchant = Merchant.find(params[:merchant_id])
-      @items = @merchant.items
+      @enabled_items = @merchant.items.where(status: 1)
+      @disabled_items = @merchant.items.where(status: 0)
```

Comment:
I think we cannot have .where in a controller. This may have to be refactored later

Reply:
Can this be a `.self` method in the items model?